styles.css

```diff
@@ -467,23 +521,28 @@ body {
     background-repeat: no-repeat;
     width: 100%;
     background-size: contain;
-    background-position-y: 100px;
+    background-position-y: 95px;
     background-position-x: 10px;
     overflow: hidden;
   }
 
   .header-Page {
     display: flex;
     flex-direction: column;
-    padding: 250px 260px 94px 260px;
+    justify-content: flex-start;
+    width: 50%;
+
+    /* padding: 250px 260px 94px 260px; */
```

Comment:
- [ ] I have noticed that you have the commented codes inside your CSS files. I kindly suggested that you remove them as many commented codes can create distractions 😞 .